packages/node_modules/cerebral/src/views/View.js

```diff
@@ -17,14 +18,25 @@ class View {
     displayName,
     onUpdate,
   }) {
+    if (typeof dependencies === 'function') {
+      throwError(
+        'You can not use a function to define dependencies. Use tags or a function on the specific property you want to dynamically create'
+      )
+    }
+
+    if (!dependencies) {
+      throwError(
+        'There is no reason to connect a component that has no dependencies'
+      )
+    }
     this.stateGetter = this.stateGetter.bind(this)
     this.signalGetter = this.signalGetter.bind(this)
     this.mergeProps = mergeProps
     this.controller = controller
     this._displayName = displayName
     this._hasWarnedBigComponent = false
-    this.onUpdate = onUpdate
-
+    this.onUpdate = onUpdate || noop
+    this._verifyPropsWarned = false
```

Comment:
@christianalfoni Each component (view) should warn one time only and it should be testable